packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -3,24 +3,34 @@ import { createFocusOutlineStyle } from '@fluentui/react-tabster';
 import { tokens } from '@fluentui/react-theme';
 import type { SliderState } from './Slider.types';
 
-export const sliderClassName = 'fui-Slider';
+export const sliderClassNames = {
```

Comment:
You should use the `SlotClassNames` helper type from `react-utilities` here:

```suggestion
export const sliderClassNames: SlotClassNames<SliderSlots> = {
```

Reply:
yup, i wrote this before the helper was checked in